ban/core/models.py

```diff
@@ -112,7 +112,7 @@ class Group(NamedModel):
 
     kind = db.CharField(max_length=64, choices=KIND)
     addressing = db.CharField(max_length=16, choices=ADDRESSING, null=True)
-    fantoir = db.CharField(max_length=9, null=True, unique=True)
+    fantoir = db.FantoirField(max_length=9, null=True, unique=True)
```

Comment:
You can keep the value inside the field. The idea was to use `resource_schema` to override the db value, because:

- we need Cerberus to allow value of length from 9 to 10
- the DB to only allow field of length 9

So:
- the peewee field must only know about `max_length=9`
- the resource schema must allow 9 and 10

Given that the resource schema is extrapoled from the peewee field, we need to override the value only for the resource schema. This can be done using `resource_schema` attribute. See for example: https://github.com/BaseAdresseNationale/ban/blob/master/ban/core/models.py#L27